### PR TITLE
edited indentations in .md files

### PR DESCRIPTION
**Pull request by TharinduNM**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: scim2/tests/test.bal [hunk 1]
```diff
@@ -4,23 +4,23 @@ import ballerina/io;
...
-                                                   trustStore:{
-                                                                  filePath:"/home/tharindu/Documents/IS_HOME/repository/resources/security/truststore.p12",
-                                                                  password:"wso2carbon"
-                                                              }
-                                               }
-                              }]
-                 }
+        auth:{
+            scheme:"oauth",
+            accessToken:"d8adc620-06b0-319f-9028-ec62fc6a7870",
```
Comment (keerthu) [inline scim2/tests/test.bal]: Can we get these values from config files instead of hard coding them?

File: README.md [hunk 2]
```diff
@@ -28,16 +28,15 @@ The following sections provide you with information on how to use the Ballerina
...
 
 
 ## Getting started
 
-1. Clone and build Ballerina from the source by following the steps given in the README.md 
-file at https://github.com/ballerina-lang/ballerina
-2. Extract the Ballerina distribution created at
- `distribution/zip/ballerina/target/ballerina-<version>-SNAPSHOT.zip` and set the 
- PATH environment variable to the bin directory.
+1. Clone the repository by running the following command
```
Comment (keerthu) [inline README.md]: As we don't need to clone the package in order to use the connector, remove it for "Getting started" section and give the reference to the source code separately.

File: scim2/Package.md [hunk 3]
```diff
@@ -27,22 +36,27 @@ import wso2/scim2;
...
-                                  secureSocket:{
-                                                   trustStore:{
-                                                                  filePath:"<......>",
-                                                                  password:"<......>"
-                                                              }
-                                               }
-                              }]
-                 }
+        auth:{
+            scheme:"oauth",
```
Comment (keerthu) [inline scim2/Package.md]: Can't we define the scheme in the connector itself as it will be always "oauth".

Comment (TharinduNM) [reply scim2/Package.md]: SCIM2 specification supports Basic Auth also. Therefore I think the user should be given the choice.

File: README.md (hunk 2 above)
Comment (keerthu) [reply README.md]: But still do we need to clone the package to use the connector? We need to import the connector from central right?

**Pull request merged**